### ISSUE TITLE
setup-python breaking in python-package-conda.yml

### ISSUE THREAD

**Issue by aph42** [actions/starter-workflows#1839]
The "Python Package using Conda" starter workflow is breaking for me out of the box on the 'Set up Python 3.10' step. I believe the issue is that there should be quotes around the python version on line 16:
`      with:
        python-version: 3.10`
should be
`      with:
        python-version: "3.10"'

As written it fails with a message about python version 3.1 not being available. Adding the quotes corrected the issue.

**Comment by Phantsure** [comment]
Yes you are write. I see that is recommended as well in setup-python repo. @aph42 Can you take this small change?
